File: HareQuestAPI/API/Dtos/ClassGroups/ClassGroupDto.cs
namespace API.Dtos.ClassGroups
{
  public class ClassGroupDto
  {
    public string Name { get; set; }
    public string ClassCode { get; set; }
    public string ClassLevel { get; set; }
    public List<ClassStudentsDto> Students { get; set; }
  }
}

File: HareQuest/HareQuest/Models/ClassGroupDto.swift
//
//  ClassGroupDto.swift
//  HareQuest
//
//  Created by Luke Hilliard on 16/04/2026.
//

import Foundation


struct ClassGroupDto: Codable {
	let teacherId: UUID
	let name: String
	let classLevel: String
}

struct ClassStudent: Codable {
	let id: UUID
	let parentId: UUID
	let studentName: String
}

struct ClassGroupResponse: Codable {
	let id: UUID
	let status: Int
	let message: String
	let success: Bool
	let teacherId: UUID
	let name: String
	let classCode: String
	let classLevel: String
	let students: [ClassStudent]?
}

